test/integration/abi-version.int.c

```diff
@@ -13,50 +14,50 @@ test_invoke (TSS2_SYS_CONTEXT *sapi_context)
     TSS2_TCTI_CONTEXT *tcti_context = NULL;
     TSS2_ABI_VERSION tstAbiVersion = { TSSWG_INTEROP, TSS_SAPI_FIRST_FAMILY, TSS_SAPI_FIRST_LEVEL, TSS_SAPI_FIRST_VERSION };
 
-    print_log( "ABI NEGOTIATION TESTS" );
+    LOG_INFO( "ABI NEGOTIATION TESTS" );
 
     // Get the size needed for sapi context structure.
     contextSize = Tss2_Sys_GetContextSize( 0 );
 
     rc = Tss2_Sys_GetTctiContext (sapi_context, &tcti_context);
     if( rc != TSS2_RC_SUCCESS )
     {
-        print_fail("ABIVersion FAILED! Response Code : %x", rc);
+        LOG_ERROR("ABIVersion FAILED! Response Code : %x", rc);
```

Comment:
The print_fail function called exit(1) internally. We will need to return 1 from here also. Otherwise the test will be reported as passed.

Reply:
I fixed them all...